From a8df2ce5f69aa65c88b63ee7836d941ad7caaf61 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Tue, 24 Jul 2018 10:42:21 -0700
Subject: [PATCH] MANTA-3849 'manta-adm update' takes a '--config-file' opt but
 does not use it Reviewed by: Kody A Kantor <kody@kkantor.com> Approved by:
 Kody A Kantor <kody@kkantor.com>

---
 cmd/manta-adm.js | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/cmd/manta-adm.js b/cmd/manta-adm.js
index f277622..5b004aa 100755
--- a/cmd/manta-adm.js
+++ b/cmd/manta-adm.js
@@ -7,7 +7,7 @@
  */
 
 /*
- * Copyright (c) 2017, Joyent, Inc.
+ * Copyright (c) 2018, Joyent, Inc.
  */
 
 /*
@@ -538,13 +538,15 @@ MantaAdm.prototype.do_update = function (subcmd, opts, args, callback)
 };
 
 MantaAdm.prototype.do_update.help =
-    'Update deployment to match a JSON configuration.\n\n{{options}}';
+    'Update deployment to match a JSON configuration.\n\n' +
+    'Usage:\n\n' +
+    '    manta-adm update [OPTIONS] CONFIG-FILE [SERVICE]\n\n' +
+    '{{options}}';
 
 MantaAdm.prototype.do_update.options = [
     maCommonOptions.logFile,
     maCommonOptions.dryrun,
     maCommonOptions.confirm,
-    maCommonOptions.configFile,
 {
     'names': [ 'no-reprovision' ],
     'type': 'bool',
-- 
2.21.0

